pkgs/development/libraries/gd/default.nix

```diff
@@ -20,13 +20,24 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
+
+  patches = [
+    # TODO: remove once dd6615fd138e53656a7883015d4a6b2d02292b26
+    #       is included in a release
+    ./libtool.patch
```

Comment:
Any chance you could use `fetchpatch` against https://github.com/libgd/libgd/commit/502e4cd873c3b37b307b9f450ef827d40916c3d6.patch here? I'm generally reluctant to bundle .patch files that are easily obtainable elsewhere in nixpkgs, as people generally forget to remove them after we stop using them and they clutter up nixpkgs.